File: java/unidad_03/Problema_3_10/src/Problema_3_10.java
/*
 * El programa calcula el término 180 de la secuencia Fibonacci.
 */

public class Problema_3_10 {

    public static void main(String[] args) {

        int primero = 0;
        int segundo = 1;

        int siguiente = 0;

        for (int i = 3; i <= 180; i++) {

            siguiente = primero + segundo;

            primero = segundo;
            segundo = siguiente;
        }
        System.out.println("El término 180 de la secuencia Fibonacci es:");

        System.out.println(siguiente);
    }
}

File: python/unidad_03/problema_3_10.py
'''
El programa calcula el término 180 de la secuencia Fibonacci.
'''

def main():

    primero = 0
    segundo = 1

    for i in range(3, 180 + 1):

        siguiente = primero + segundo

        primero = segundo
        segundo = siguiente

    print("El término 180 de la secuencia Fibonacci es:")

    print(siguiente)

if __name__ == '__main__':

    main()